src/test/java/com/avp/homework7objectsinmemory/task4tvandtvconroller/service/TvControllerServiceTest.java

```diff
@@ -0,0 +1,76 @@
+package com.avp.homework7objectsinmemory.task4tvandtvconroller.service;
+
+import com.avp.homework7objectsinmemory.task4tvandtvconroller.bean.TvController;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TvControllerServiceTest {
+
+    private static final Logger log = LoggerFactory.getLogger(TvControllerServiceTest.class);
+    private TvController testTvController;
+    private TvControllerService test;
+
+    @BeforeEach
+    void beforeEach() {
+        testTvController = new TvController();
+        test = new TvControllerService(testTvController);
+        testTvController.setCurrentChannel(3);
+        testTvController.setCurrentSound(15);
+    }
+
+    @Test
+    void IncreaseSound() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
I agree I didn't notice